MUXSDKImaListener/Classes/MuxImaListener.h

```diff
@@ -10,18 +10,19 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @import MuxCore;
+@import MUXSDKStats;
 @import GoogleInteractiveMediaAds;
 
 @class MUXSDKPlayerBinding;
 
 @interface MuxImaListener : NSObject {
-    MUXSDKPlayerBinding *_playerBinding;
+    @public MUXSDKPlayerBinding *_playerBinding;
```

Comment:
the `_` prefix in `_playerBinding` is a common convention for privates. make it `playerBinding` if it needs to be public otherwise mark it private